.babelrc

```diff
@@ -4,14 +4,17 @@
       "pragma": "createElement"
     }],
     ["@babel/preset-env", {
+       "bugfixes": true,
        "targets": {
-         "browsers": ["> 0.3%"]
+         "chrome": "55",
+         "edge": "17",
+         "firefox": "53",
+         "safari": "10.1",
       }
     }]
   ],
   "plugins": [
-    "angularjs-annotate",
-    "transform-async-to-promises"
+    "angularjs-annotate"
```

Comment:
I happen to know why we use this plugin, but, man, I feel like it could be confusing for future devs. Do we have any options for migrating away from this for our service injections?

Reply:
Yes, we could probably create our own plugin which does the same thing but avoids the Angular reference. Coincidentally I had to explain this vestige to Eduardo earlier today.